.github/workflows/cron.yml

```diff
@@ -50,14 +51,19 @@ jobs:
       run: |
         pytest
       shell: bash
+    - name: generate datalog
+      if: steps.new-commits.outputs.NEW_COMMITS == '1'
```

Comment:
so, if more than one commit in MR then it should be ignored?

Reply:
This is the Cron pipeline - it won't run on PRs - and what it does is using git porcelain to check if there are local uncommitted changes - and in case there are these step should be run

Reply:
ah, indeed, nice